app/models/player.rb

```diff
@@ -44,6 +44,9 @@ def sos_earned
 
   def side_bias
     @side_bias ||= pairings.reported.reduce(0) do |bias, pairing|
```

Comment:
you probably want something along the lines of `pairings.includes(:stages).reported.reduce(0)` so that is retrieved as part of the same query.